includes/Core/Authentication/Authentication.php

```diff
@@ -443,6 +451,47 @@ public function get_disconnect_url() {
 		);
 	}
 
+	/**
+	 * Gets the proxy permission url
```

Comment:
_All_ documentation should end in a period. (unless it already has exclamation mark, question mark etc of course). Also, let's use correct capitalization of abbreviations.